logrotate.conf

```diff
@@ -0,0 +1,8 @@
+/var/log/nginx/*.log /var/log/consul-template/*.log {
```

Comment:
This should be /var/log/nginx/log

that's the path that OpenLMIS uses for log files

your path is symbolic links

Not sure why it is set up this way tbh, but that's the OpenLMIS setup